pkg/helpers/version/version.go

```diff
@@ -0,0 +1,42 @@
+// Copyright Contributors to the Open Cluster Management project
+package version
+
+import (
+	"errors"
+)
+
+type VersionBundle struct {	
+	Registration string
+	Placement string
+}	
+
+
```

Comment:
how about we have

```go
var versionBundle = map[string]VersionBundle{
  "default": ...
  "0.5.0":
}
```

Reply:
to which service "0.5.0" is referring too? 

how do you bundle 
work 0.4.0 
placement 0.6.0?

Reply:
```
var versionBundle = map[string]VersionBundle{
  "default": ...
  "0.5.0": VersionBundle{
      "work": "0.5.0"
       "placement": "0.3.0",
       "registration": "0.5.0",
       "operator": "0.5.0",
   },
}
```

Reply:
or i mean you still have this GetVersionBundle func, but the map in it could be written a var a priori, just for the ease of code reading for other developers :)